docs/read-from-stream-layer.md

```diff
@@ -134,9 +134,9 @@ You can read messages from a [stream layer](https://www.here.com/docs/bundle/dat
 
 1. Create the `streamLayerClient` object.
 
-   For instructions, see [Create StreamLayerClient]([#create-streamlayerclient](https://www.here.com/docs/bundle/data-api-developer-guide/page/rest/layer-creating_2.html)).
+   For instructions, see [Create StreamLayerClient](#create-streamlayerclient).
 
-2. Subscribe to the stream layer. See [Subscribe to the stream layer](https://www.here.com/docs/bundle/data-api-developer-guide/page/rest/getting-data-stream.html)
+2. Subscribe to the stream layer.
```

Comment:
There's 'Subscribe to a stream layer' section in this file. Could be that section is useful for users. I don't have background in the area this file documenting.